Exercicio3/src/funcional/3a.ts

```diff
@@ -0,0 +1,41 @@
+function isNumber(array: Array<any>): boolean { // validar se o array e numerico
+    let validation: boolean = true;
+    array.forEach(el => {
```

Comment:
não usar aberviações em nomes de variavéis. modificar o nome `el` para outro descritivo